test/spec/utils/alt-resolver.test.js

```diff
@@ -52,7 +54,8 @@ describe('Alt Resolver', () => {
 
   it('should render 500 on error', (done) => {
     (async function () {
-      const content = await altResolver.render(DummyError, flux, true);
+      const Stubbed = stubApp(flux)(DummyError);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).